generators/go/internal/generator/sdk/core/stream.go

```diff
@@ -162,11 +162,15 @@ func (s *scannerStreamReader) parse(bytes []byte) (int, []byte, error) {
 		}
 	}
 	data := bytes[start:]
-	if i := strings.Index(string(data), s.options.delimiter); i >= 0 {
-		data = data[:i+len(s.options.delimiter)]
+	delimIndex := strings.Index(string(data), s.options.delimiter)
+	if delimIndex < 0 {
+		return len(bytes), data, nil
```

Comment:
Also, I think `n` should be set to `start + len(data)` here - `len(bytes)` won't cut it since we need to consume the prefix, too.